scripts/template_upload_ec2.py

```diff
@@ -225,12 +247,16 @@ def run(**kwargs):
      """
     mgmt_sys = cfme_data['management_systems']
```

Comment:
Why not just say
```python
if 'ec2east' in cfme_data['management_systems']:
    ssh_rhevm......
```
seems like you are hardcoding it anyway and only using one, there is no need to loop here.